// Copyright (c) Microsoft Corporation.
// Licensed under the MIT license.

syntax = "proto2";


message t_bsr_stats {
   required uint32 ue_index = 1;
   required uint32 cnt = 2;
}

message bsr_stats {
   required uint64 timestamp = 1;
   repeated t_bsr_stats stats = 2;
}





